### DISCUSSION TITLE
How to setting gird interval use empty line？

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#4646] [Q&A]
I use a grid layout and I want every row of Gird to be arranged in a compact and orderly manner. Which CSS attribute should I use?
Empty space between elements in a row

![image](https://github.com/Textualize/textual/assets/34232854/6ec6e7cc-46e0-4661-98b6-2f08d77fe6ff)

**Comment by TomJGooding** [comment]
Sorry I'm not sure I understand the question. From the screenshot above you've clearly managed to add some spacing already, so what is stopping you from just replicating that?

**Reply by Yurii-huang** [reply]
This is the spacing automatically set by the grid based on height, but I don't like setting the spacing between components to ensure that each row only occupies one row.
Is it impossible to do this?

**Reply by Yurii-huang** [reply]
I don't want to change the row coordinates of the original components based on the height of the grid. What should I do？

**Comment by Yurii-huang** [comment]
use grid-gutter
